wrapper_test.go

```diff
@@ -280,16 +280,21 @@ func (w *wrapperTestSuite) TestLambdaHandlerE2ELocal() {
 			assert.Equal(w.T(), "bd862e3fe1be46a994272793", lumigoStart.TransactionID)
 			assert.Equal(w.T(), string(inputPayload), lumigoStart.Event)
 			assert.Equal(w.T(), version, lumigoStart.SpanInfo.TracerVersion.Version)
-			if lumigoStart.LambdaType == "http" {
-				assert.NotNil(w.T(), lumigoStart.SpanInfo.HttpInfo)
-				assert.Equal(w.T(), ts.URL, fmt.Sprintf("http://%s", lumigoStart.SpanInfo.HttpInfo.Host))
-				assert.Equal(w.T(), fmt.Sprintf("%s/", ts.URL), lumigoStart.SpanInfo.HttpInfo.Request.URI)
-				assert.Equal(w.T(), "POST", lumigoStart.SpanInfo.HttpInfo.Request.Method)
-				assert.Equal(w.T(), `{\"name\": \"test\"}`, lumigoStart.SpanInfo.HttpInfo.Request.Body)
-				assert.Contains(w.T(), `"Agent": "test"`, lumigoStart.SpanInfo.HttpInfo.Request.Headers)
+
+			if len(spans.endSpan) > 1 {
```

Comment:
What is the structure of `endSpan`? Does it hold the http-spans?
The end-span isn't suppose to hold the http-spans, it's a span with the data about the end of the invocation (for example, return-value)